pkg/controller/datavolume/clone-controller-base.go

```diff
@@ -437,7 +437,10 @@ func (r *CloneReconcilerBase) updateStatusPhaseForPopulator(pvc *corev1.Persiste
 		//dataVolumeCopy.Status.Phase = cdiv1.Unknown // hold off on this for now
 		return nil
 	}
-	dataVolumeCopy.Status.Phase = dvPhase
+	// Avoid setting DV to failed for consistency with non-populator flow
+	if dvPhase != cdiv1.Failed {
+		dataVolumeCopy.Status.Phase = dvPhase
+	}
```

Comment:
Not sure I understand why not use the `Failed` phase, as there are other phases (e.g. `RebindInProgress`) not used in the non-populator flow afaik.

Reply:
That's true, however, the populator-specific phases are transitory so they don't mess with tests/flow that much. Adding a failed phase at the end seems kinda harsh and (most importantly) breaks consistency with DataVolume behavior in non-populator flow.